__tests__/db.test.js

```diff
@@ -56,4 +112,83 @@ describe("GET /api", () =>{
             })
         })
     })
+})
+
+describe("POST /api/articles/:article_id/comments", () =>{
+    test("Returns a 201 when posting data", ()=>{
+        return request(app).post("/api/articles/1/comments").send({username:"lurker", body:"Sorry -snip-"})
+            .then((res)=>{
+                expect(res.statusCode).toBe(201)
+        })
+    })
+    test("Returns only the comment when successfully posted", ()=>{
+        return request(app).post("/api/articles/1/comments").send({username:"lurker", body:"Sorry -snip-"})
+            .then((res)=>{
+                expect(res.statusCode).toBe(201)
+                expect(res.body.postedComment).toBe("Sorry -snip-")
+                expect(Object.keys(res.body)).toEqual(["postedComment"])
```

Comment:
Realistically, you should be getting back a comment_id, the username, the article_id etc - everything should be sent back